workspace/apps/pidp/src/app/features/profile/pages/account-linking/account-linking.page.scss

```diff
@@ -211,6 +211,12 @@
       .idir-account {
         margin-top: 1.19rem;
       }
+
+      .card-hover:hover {
+        background-color: #ebebeb;
```

Comment:
I've noticed the colour `#ebebeb` being used in a couple new places. Just for reference, If we can re-use one of the existing colours in `_pidp-variable.scss` before creating a new colour scheme variable. For consistency purposes, we would like to maintain the same look across the app.

Reply:
for example this colour has a near small delta
<img width="787" alt="delta 1 47" src="https://github.com/user-attachments/assets/b5c0f336-3962-4d91-a8bf-8a2bf3c88dda" />